api/v1alpha1/volumereplicationgroup_types.go

```diff
@@ -159,17 +159,35 @@ type VolumeReplicationGroupSpec struct {
 	KubeObjectProtection *KubeObjectProtectionSpec `json:"kubeObjectProtection,omitempty"`
 }
 
+const ReservedBackupName = "use-backup-not-restore"
+
+type RecipeSpec struct {
+	//+optional
+	Name *string `json:"name,omitempty"`
+
+	//+optional
+	Workflow *WorkflowSpec `json:"workflow,omitempty"`
+}
+
+type WorkflowSpec struct {
+	//+optional
+	CaptureName *string `json:"captureName,omitempty"`
+
+	//+optional
+	RecoverName *string `json:"recoverName,omitempty"`
+
+	//+optional
+	VolumeGroupName *string `json:"volumeGroupName,omitempty"`
```

Comment:
If I'm understanding the comment correctly, I think it may be worth a discussion on how to handle multiple labels in general. Benamar raised a [similar issue](https://github.com/RamenDR/ramen/pull/675#discussion_r1108519905), but about how to combine pvcLabelSelector labels from the VRG and the Volume Group LabelSelectors.

Reply:
I think moving this as a label selector would be a start, to write more complex label selections.